patches/model-pivot.yml

```diff
@@ -0,0 +1,8 @@
+Add:
+    Model:
+      WorldPivotData:
+        Serialization:
+          Type: Serializes
+        DataType:
+          Value: OptionalCFrame
+        Scriptability: Custom
```

Comment:
`WorldPivotData` is kind of a funny name. That's what this property serializes as, but we should surface it as `WorldPivot` to the user. There is a little bit of guidance in this document that might help: https://dom.rojo.space/patching-database.html

You can also look at the patches in `instance.yml` that add support for attributes. They serialize as `AttributesSerialize` but we expose them as `Attributes`.

Reply:
I made it use an alias like you suggested in the latest commit.